sdk/tests/deqp/functional/gles3/es3fPixelBufferObjectTest.js

```diff
@@ -425,8 +425,16 @@ var tcuImageCompare = framework.common.tcuImageCompare;
 
         var isOk = false;
 
-        if (floatCompare)
-            isOk = tcuImageCompare.floatThresholdCompare('Result comparison', 'Result of read pixels to memory compared with result of read pixels to buffer', readReference.getLevel(0), readResult, [0.0, 0.0, 0.0, 0.0]);
+        if (floatCompare) {
+            // The result of rgb10_a2 from Intel GPU would have slight difference with the one from CPU, so some tolerance is added here.
+            // Detailed discussion in Mesa upstream can be found at https://bugs.freedesktop.org/show_bug.cgi?id=89314.
+            var threshold;
+            if (tcuTestCase.runner.currentTest.name == 'rgb10_a2_triangles')
```

Comment:
Is there any more robust way to check for this aside from looking at the test's name? Or is it only this exact test (and not the clear tests, for example, in this file) that need this workaround?